setup.cfg

```
@@ -105,30 +101,27 @@ universal = 1
 pins =
     ###
     # mutually compatible pyviz-maintained packages
-    bokeh = 1.0.0dev6
-    holoviews = 1.11.0a8
-    hvplot = 0.2.1
-    geoviews = 1.5.4a13
-    datashader = 0.6.8
-    panel = 0.1.2
-    param = 1.8.1
-    paramnb = 2.0.4
-    parambokeh = 0.2.3
+    bokeh = 1.0.4
+    holoviews = 1.11.2
+    hvplot = 0.4.0
+    geoviews = 1.6.2
+    datashader = 0.6.9
+    panel = 0.4.0
+    param = 1.8.2
     ###
     # avoid tab-completion issue in ipython=6
     ipython = 5.*
     # for datashader compatibility
-    dask = 0.18.2
+    dask = 1.1.0
     # StreamingDataFrame name change in 0.3.0
-    streamz = 0.3.0
-    ipywidgets = 7.*
+    streamz = 0.5.0
     # for jupyter notebook and bokeh server
     tornado = 4.5.3
     # in case of unknown future or past compatibility issues
     python = 3.6.*
     numpy = 1.14.5
-    pandas = 0.23.4
-    xarray = 0.10.3
+    pandas = 0.24.0
+    xarray = 0.11.0
```

It'd be nice to have 0.11.2 if it's available - adds some nice features for netcdf - but it's not on defaults yet, so 
could we just have a floor pin?

```suggestion
    xarray >= 0.11.0
```

---

I'm actually not totally clear what this section is about, I thought this was only for running the tests where I'd prefer a fixed pin.

---

The pins section is used by pyctdev when `--pin-deps` is passed. 

`--pin-deps` was being used for running the tests and for generating packages (i.e. to generate highly pinned packages).

Using it for the tests was removed in #152. I don't know if you are planning to continue to release highly pinned packages or not.

---

Thanks Chris, it's all starting to fit together for me. I'll pin 0.11.2 now that I've added conda-forge.